docs/operator/README.md

```diff
@@ -2,10 +2,6 @@
 sort: 0
 weight: 0
 title: VictoriaMetrics Operator
-menu:
```

Comment:
@tenmozes @Amper is this change expected?

Reply:
Yes